README.md

```diff
@@ -111,11 +111,46 @@ conforming to MEDS binary classification prediction schema:
 ./MEDS-DEV/src/MEDS_DEV/helpers/generate_predictions.sh $MEDS_ROOT_DIR $TASK_NAME
 ```
 
+In order to work with the evaluation package (see the next section),
+the model's outputs must conform to the _prediction schema_:
+
+```python
+prediction = pa.schema(
+    [
+        ("subject_id", pa.int64()),
+        ("prediction_time", pa.timestamp("us")),
+        ("boolean_value", pa.bool_()),
+        ("predicted_boolean_value", pa.bool_()),
+        ("predicted_boolean_probability", pa.float64()),
```

Comment:
Does this need to be between 0 and 1, or can it be an unnormalized score? I know you say 0 - 1 later, but it wasn't clear to me if that was a hard requirement.

Reply:
I guess that's what I'm not sure about myself: generally my opinion would be to separate these two interpretations into their own fields, so that there is no confusion about the same field representing different concepts with different requirements; however I think at some point we decided in a discussion with @EthanSteinberg that these should be the same.

Do you have any thoughts on this @mmcdermott?

Reply:
My current feeling is to leave it with just one field and make it actual probabilities. It is not a super significant ask to have people give true probabilities if their normal output is scores, and it lets us also analyze things like calibration (in the future, not imminently) and keeps the field burden low.

Reply:
I would just have these be actual probabilities.